prover/src/lib.rs

```diff
@@ -487,6 +478,33 @@ pub trait Prover {
         Ok(proof)
     }
 
+    // HELPER METHODS
+    // --------------------------------------------------------------------------------------------
+
+    fn build_main_trace_and_commit<E>(
```

Comment:
This method will appear in the public interface - so, we should document it (similar to the `build_constraint_commitment()` method below).

Reply:
I had added that helper function when we had 2 `prove()` methods, but it's no longer needed, so I removed it